### PR TITLE
Divider component from lab to core

### PR DESCRIPTION
**Pull request by Vineet119**

### CODE REVIEW

File: site/docs/components/divider/usage.mdx [hunk 1]
```diff
@@ -20,14 +20,14 @@ data:
 
 ## Import
 
-To import `Divider` from the lab Salt package, use:
+To import `Divider` from the Core Salt package, use:
```
Comment (joshwooding) [inline site/docs/components/divider/usage.mdx:23]:
```suggestion
To import `Divider` from the core Salt package, use:
```

File: packages/core/stories/divider/divider.stories.tsx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import { StoryFn } from "@storybook/react";
 import { StackLayout } from "@salt-ds/core";
 
-import { Divider, DividerProps } from "@salt-ds/lab";
+import { Divider, DividerProps } from "@salt-ds/core";
```
Comment (joshwooding) [inline packages/core/stories/divider/divider.stories.tsx:4]: This should be merged into the import above

Comment (Vineet119) [reply packages/core/stories/divider/divider.stories.tsx]: done

File: site/docs/components/divider/usage.mdx (hunk 1 above)
Comment (Vineet119) [reply site/docs/components/divider/usage.mdx]: done

**Review by joshwooding — APPROVED** [review]

**Comment by origami-z** [comment]
Spoke offline, let's update some existing examples (e.g. Dropdown select all, Ag grid custom status bar) to use Divider component to test the water.

**Comment by joshwooding** [comment]
> Spoke offline, let's update some existing examples (e.g. Dropdown select all, Ag grid custom status bar) to use Divider component to test the water.

Can we do that in a different PR? I don't like when promotion to core PRs have too many additional changes

**Review by joshwooding — APPROVED** [review]

**Pull request merged**